File: QuestoesBeecrowd-Iniciante/1010.c
#include <stdio.h>
#include <stdlib.h>

float valor(int a2, float a3, int b2, float b3)
{
    float custo;
    custo = (a2 * a3) + (b2 * b3);
    return (custo);
}

int main()
{
    int cod1, qt1, cod2, qt2;
    float preco1, preco2, preco;
    scanf("%d %d %f", &cod1, &qt1, &preco1);

    scanf("%d %d %f", &cod2, &qt2, &preco2);

    preco = valor(qt1, preco1, qt2, preco2);
    printf("VALOR A PAGAR: R$ %.2f\n", preco);

    return 0;
}

File: src/Iniciante/1010.java
package Iniciante;

import java.io.IOException;
import java.util.Scanner;

import javax.print.DocFlavor.STRING;

import java.util.Locale;

public class Main {
    public static void main(String[] args) throws IOException {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        int cod1 = input.nextInt();
        int qt1 = input.nextInt();
        double valor = input.nextDouble();
        int cod2 = input.nextInt();
        int qt2 = input.nextInt();
        double valor2 = input.nextDouble();
        double soma = qt1*valor + qt2 *valor2;
        String total = String.format("%.2f", soma);
        System.out.println("VALOR A PAGAR: R$ "+total);


        input.close();
        
    }
    

}
